src/main/scala/shade/memcached/internals/SpyMemcachedIntegration.scala

```diff
@@ -436,6 +468,8 @@ class SpyMemcachedIntegration(cf: ConnectionFactory, addrs: Seq[InetSocketAddres
         case CASResponse.OBSERVE_TIMEOUT =>
           CASObserveTimeout
       }
+    case x if x.isSuccess =>
```

Comment:
Most of the spymemcached operations return some subclass of CASOperationStatus, even if the operation isn't a CAS. For example, see [GetOperationImpl] (https://github.com/couchbase/spymemcached/blob/master/src/main/java/net/spy/memcached/protocol/binary/GetOperationImpl.java#L56) and its superclass [OperationImpl] (https://github.com/couchbase/spymemcached/blob/master/src/main/java/net/spy/memcached/protocol/binary/OperationImpl.java#L73).

[MutatorOperationImpl] (https://github.com/couchbase/spymemcached/blob/master/src/main/java/net/spy/memcached/protocol/binary/MutatorOperationImpl.java#L70) does not however, returning an instance of the superclass OperationStatus , and so the existing case does not match.